rest_api/javascript/click_to_queue.html

```diff
@@ -0,0 +1,79 @@
+<div>
+  <div class='queue' queue_id='a6d8feb6-c300-4409-9e8a-c9842648c6ef'>
+    <button>TEAM A</button>
+  </div>
+  <div class='queue' queue_id='df94560d-af53-4836-a83d-1c405e291c0a'>
+    <button>TEAM B</button>
+  </div>
+</div>
+<script>
+  function findQueueElement(queueId) {
+    return document.querySelector("div.queue[queue_id='" + queueId + "']");
+  }
+  function findAllQueueElements() {
+    return document.querySelectorAll('div.queue');
+  }
+  function showCanQueue(queueElement, queueMedias) {
+    queueElement.querySelector('button').disabled = false;
+  }
+  function showCannotQueue(queueElement) {
+    queueElement.querySelector('button').disabled = true;
+  }
+  // Handle Queue state changes for a particular queue.
+  // Enable queuing and media buttons for available media if open, disable
+  // otherwise.
+  function onQueueState(queue) {
+    console.log(queue);
+    if (findQueueElement(queue.id) === null) {
+      // Queue not related to the current page, ignore
+    } else if (queue.state.status === queue.state.STATUSES.OPEN) {
+      showCanQueue(findQueueElement(queue.id), queue.state.medias);
+    } else {
+      showCannotQueue(findQueueElement(queue.id));
+    }
+  }
+  function showCannotQueueAnywhere() {
+    findAllQueueElements().forEach(showCannotQueue);
+  }
+  function findAllQueueMediaButtons() {
+    return document.querySelectorAll('div.queue > button')
```

Comment:
There are no media buttons in the example HTML, only team buttons which makes this confusing

Reply:
Added two media buttons, but these files are just examples